### PR TITLE
Created most of the front end for the deviations page

### PR DESCRIPTION
**Pull request by shhmon**
This resulted in some fiddling with js and css files for the base-schedule page aswell. You can now add deviations and remove them. The date picker is temporary.

### CODE REVIEW

**Comment by Greenheart** [comment]
Looking and working good! :smile: 

### Some feedback:
- [ ] I think the `add`\- and `remove`-buttons need to have a bigger clickable area.
- [x] I found some javascript-linting-errors. (Check diff-comments below)

Fix these things, do an interactive rebase to squash commits (`git rebase -i HEAD~<number of commits>`) and then push back up again

File: web/app/static/js/main/slider-config.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    // Work period 2 can't be longer than 5 hours
+    if (ui.values[3] - ui.values[2] > 300) {
+      $(jQuery(this).children('.ui-slider-range.return')).addClass('errorbg')
+    } else {
+      $(jQuery(this).children('.ui-slider-range.return')).removeClass('errorbg')
+    }
+  },
+  change: function (e, ui) {
+    var data = { 'values' : ui.values }
```
Comment (Greenheart) [inline web/app/static/js/main/slider-config.js]: ###### Extra space after key "values".at line 52 col 6

Change `{ 'values' : ui.values }` to `{ 'values: ui.values' }`

File: web/app/static/js/main/deviations.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// code specific to the UI in the `/deviations`-route
+
+// Maybe move the slider-init-code from create-slider to here?
+
+$(document).ready(function () {
+
```
Comment (Greenheart) [inline web/app/static/js/main/deviations.js]: ###### Block must not be padded by blank lines.at line 5 col 2

Remove line 6

File: web/app/static/js/main/deviations.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    // Find and initiate the slider in the new cloned object
+    var slider = deviation.find('.period-slider')
+    $(slider).slider(jQuery.extend(true, {}, options))
+
+    deviation.find('.remove-deviation').click(removeDeviation)
+
+    $('.buttons').before(deviation)
+  })
+
+  function removeDeviation(e) {
```
Comment (Greenheart) [inline web/app/static/js/main/deviations.js]: ###### Missing space before function parentheses.at line 26 col 3

By adding spaces like this to function definitions, it's easier to separate funcNames from their parameters

**Comment by shhmon** [comment]
@Greenheart I've corrected the errors. The area of the buttons didn't strike me as a milestone 0 improvement, so I won't fiddle with that just yet.

Edit: Created issue #140 for the button area question.

**Comment by Limpan** [comment]
Amend your latest commit and force push to your fork and the pull request will update automatically.

**Comment by Greenheart** [comment]
@Catagen Good job!

**Pull request merged**